static/js/hoc/withDoubleColumn.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import React from "react"
+import R from "ramda"
+
+import ChannelSidebar from "../components/ChannelSidebar"
+import Sidebar from "../components/Sidebar"
+
+const withDoubleColumn = R.curry(
+  (className: string, WrappedComponent: Class<React.Component<*, *>>) => {
```

Comment:
Looks like className isn't actually used in this HOC. Am I missing something here? Might just be out of practice with react...